phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -154,6 +143,8 @@
     private int  singleRowRebuildReturnCode;
     private Map<byte[], NavigableSet<byte[]>> familyMap;
     private byte[][] viewConstants;
+    private IndexVerificationResultRepository verificationResultRepository;
```

Comment:
How these two repositories differ from each other? Could the names be more explicit?

Reply:
Each repo controls insert/select operations for a particular table: the result table (PHOENIX_INDEX_TOOL_RESULT) that stores summary information about the run (counters, etc), and the output table (PHOENIX_INDEX_TOOL) that contains the row-level information about the failed rows. 

Please let me know if you have suggestions on clearer names. 

(The word "Repository", btw, is the name of the design pattern being used -- https://martinfowler.com/eaaCatalog/repository.html , from Martin Fowler's Patterns of Enterprise Architecture and Eric Evans' Domain Driven Design)